package.json

```diff
@@ -102,15 +97,14 @@
         "text": "node_modules/requirejs-text/text",
         "jquery": "node_modules/jquery/dist/jquery.min",
         "js-cookie": "node_modules/js-cookie/src/js.cookie",
-        "select2": "node_modules/select2/select2",
         "select-woo": "node_modules/select-woo/dist/js/selectWoo.full",
         "select-woo-src": "node_modules/select-woo/src/js/select2",
         "bootstrap": "node_modules/bootstrap/dist/js/bootstrap.min",
         "jquery-ui": "node_modules/jqueryui/jquery-ui.min",
         "backbone": "node_modules/backbone/backbone-min",
         "underscore": "node_modules/underscore/underscore-min",
         "jquery-validate": "node_modules/jquery-validation/dist/jquery.validate.min",
-        "d3": "node_modules/d3/dist/d3.min",
+        "d3": "node_modules/d3/src/index.js",
```

Comment:
I'm unable to build the frontend unless I revert this change. Do you know why?

Reply:
hmm just checked this and it's working for me -- and breaks the other way. Did you update the project's `package.json` to point to this branch?

```
    "dependencies": {
        "arches": "archesproject/arches#cbyrd-9282-convert-from-yarn-to-npm"
    },
```

Reply:
whoops, I definitely missed that--lgtm!